driver-adapters-wasm/neon-vercel-nextjs-pages-edgefn/next.config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
+  webpack(config) {
+    config.experiments ??= {}
+
+    config.experiments.asyncWebAssembly = true
```

Comment:
Seeing this again, this is definitily a smell and potential problem.
Let's open an EA issue.

Reply:
https://github.com/prisma/team-orm/issues/677

Reply:
Could it be helpful to share with users that this is just Webpack's equivalent of Node.js' `--experimental-wasm-modules`, or Vite's `vite-plugin-wasm`? For self-preservation, I think it's important to get the point across to users: these extra configurations steps required to run Prisma on Edge Functions aren't due to bad DX in Prisma itself